app/models/solidus_paypal_braintree/source.rb

```diff
@@ -33,6 +32,29 @@ def imported
       false
     end
 
+    def last_4
+      return self[:last_4] if self[:last_4]
+
+      if braintree_payment_method
+        self[:last_4] = braintree_payment_method.last_4
+        save!
+      end
+
+      self[:last_4]
+    end
+    alias_method :last_digits, :last_4
```

Comment:
You moved that one from the top to here. I think, it better to have those alias_methods grouped together.

Reply:
i did